Makefile

```diff
@@ -276,7 +281,7 @@ test-only: all  ## For a quick test, does not run linter or build docs.
 	$(MAKE) build-addons
 	$(MAKE) build-addons-napi
 	$(MAKE) cctest
-	$(MAKE) jstest
+	$(MAKE) test-js
```

Comment:
Would it not be better to remove the addon parts from this task and keep `jstest`? Just the `cctest` would have to move below. The same for the other entries.

Reply:
This make rule is designed to run a number of steps serially.  Depending on `jstest` would enable make to run these commands in parallel; and in particular, before `all` is completed.  Note that the definition of `build-addons` does not require `$(NODE_EXE)` to be present.

```
build-addons: | $(NODE_EXE) test/addons/.buildstamp
```